apps/publisher/modules/lifecycle/lifecycle-api.js

```diff
@@ -316,6 +374,7 @@ var error = '';
         var lifecycle = lcApi.getLifecycle(lcName, tenantId);
         var rxtManager = coreApi.rxtManager(tenantId);
         var lcState = am.getLifecycleState(asset, lcName);
+		var lcCheckedStates = am.getLifecycleCheckedState(asset.id, lcName);
```

Comment:
Fixed in commit 86e200939329fb59ede32b4bcdea56a8a9db2abd [1]

[1] https://github.com/sameerak/carbon-store/commit/86e200939329fb59ede32b4bcdea56a8a9db2abd